platzky/engine.py

```
@@ -25,13 +27,20 @@ def __init__(self, config: Config, db, import_name):
             default_translation_directories=babel_translation_directories,
         )
 
+        self.cms_modules: List[CmsModule] = []
+        # TODO add plugins as CMS Module - all plugins should be visible from admin page at least as configuration
```

Ruff (0.11.9)

Line too long (114 > 100)

(E501)